src/components/Map.js

```diff
@@ -49,8 +50,11 @@ export const Map = ({
         center={centerMap}
         onDragEnd={(event) => handleCenterMoved(event)}
       >
-        {locations.length > 0 && loaded
-          ? locations.map((location) => {
+        {navigator?.geolocation && userLocationShared ? (
```

Comment:
Is this where @segdeha suggested removing the `?` so it reads `navigator.geolocation` ?

Reply:
Thanks, Jessica! I am going to change `navigator?.geolocation` to `navigator.geolocation`.

Reply:
I just changed it!